lists/gb/gb-chc.json

```diff
@@ -24,32 +24,44 @@
   "listType": "primary",
   "access": {
     "availableOnline": true,
-    "onlineAccessDetails": "",
-    "publicDatabase": "https://www.gov.uk/government/organisations/charity-commission",
-    "guidanceOnLocatingIds": "",
+    "onlineAccessDetails": "Information about the data and information available through the search is available available at https://register-of-charities.charitycommission.gov.uk'. Click through to \"Find a charity\" for a simple text search of a charity's name. Use the advanced search to filter the search across multiple different aspects of the dataset.",

```

Comment:
Typo: "available available"

Reply:
That's because it's *very* available ;0)